src/main/java/uk/gov/companieshouse/company/profile/service/CompanyProfileService.java

```diff
@@ -33,4 +34,19 @@ public CompanyProfileService(Logger logger, CompanyProfileRepository companyProf
                 companyNumber, companyProfileDao));
         return companyProfileDao;
     }
+
+    /**
+     * Updated insolvency links in company profile.
+     * @param companyProfileRequest company Profile information {@link CompanyProfile}
+     */
+    public void updateInsolvencyLink(final CompanyProfile companyProfileRequest) {
+        CompanyProfileDao companyProfile = companyProfileRepository
```

Comment:
Probably CompanyProfileDocument is more appropriate name for this object than CompanyProfileDao, also the package should be model instead of domain to keep it consistent in our projects if you agree?